### ISSUE TITLE
White background on Project Board

### ISSUE THREAD

**Issue by michaelsync** [aalvarado/JiraDarkTheme#29]
![image](https://user-images.githubusercontent.com/192911/121276271-b4360c00-c911-11eb-8b8d-3ff2bb654386.png)

There are some missing DIV that need the dark theme. E.g. ReactVirtualized__Grid__innerScrollContainer

```
<div class="ReactVirtualized__Grid__innerScrollContainer" role="rowgroup" style="width: auto; height: 40px; max-width: 260px; max-height: unset; overflow: visible; position: relative; will-change: initial; background-color: rgb(244, 245, 247); transition: height 0.2s ease 0s; min-height: 100%;"><div style="height: 40px; left: 0px; position: absolute; top: 0px; width: 100%;"><div class="sc-fyjhYU dKFiUS"><div class="sc-fgfRvd bZIiNK"><div class="sc-hIVACf eGfSJn"><button class="css-jr8afv" data-testid="platform-inline-card-create.ui.trigger.visible.button" type="button" tabindex="0"><span class="css-1ujqpe8"><span class="sc-iAyFgw ghHVIH" role="presentation"><svg width="24" height="24" viewBox="0 0 24 24" focusable="false" role="presentation"><path d="M13 11V3.993A.997.997 0 0 0 12 3c-.556 0-1 .445-1 .993V11H3.993A.997.997 0 0 0 3 12c0 .557.445 1 .993 1H11v7.007c0 .548.448.993 1 .993.556 0 1-.445 1-.993V13h7.007A.997.997 0 0 0 21 12c0-.556-.445-1-.993-1H13z" fill="currentColor" fill-rule="evenodd"></path></svg></span></span><span class="css-19r5em7"><div class="sc-kQsIoO yFqwh">Create issue</div></span></button></div></div></div></div></div>

**Comment by aalvarado** [comment]
Can I get some steps to see how the project board can look like this? 

is this the same issue as? https://github.com/aalvarado/JiraDarkTheme/issues/19

**Comment by michaelsync** [comment]
Thanks, Aalvarado. Yes. It's the same issue. I will see if i can override this style "ReactVirtualized__Grid__innerScrollContainer"

**Comment by aalvarado** [comment]
Possibly resolved by #32 . Please reopen if not :)
